backend/lib/endpoints/posts.js

```diff
@@ -98,6 +100,52 @@ async function routes(app) {
       return updatedPost;
     },
   );
+
+  app.put(
+    "/:postId/likes/:userId",
```

Comment:
I guess maybe I need to rename either the schema or the endpoint because I changed schema to `"likedBy"` for clarity and now it doesn't match the route anymore

Reply:
By not matching the route, do you mean when generating the schema or by fastify router?

Reply:
@shivanshs9 I was referring to the name `/likes` compared to the field `/likedBy` ... and it also returns both `likes` and `likesCount` but I guess that's ok, updated now